new/LIT7094PrayerDevils.xml

```diff
@@ -48,6 +48,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <langUsage>
                 <language ident="en">English</language>
                 <language ident="gez">Gǝʿǝz</language>
+                <language ident="gez-tr">Gǝʿǝz transliterated</language>
```

Comment:
Where do you use this?

Reply:
I used `<foreign xml:lang="gez-tr">ʾasmāt</foreign>` in relation desc.